### PR TITLE
Exposed NavigationDrawerController methods for objc

### PR DESCRIPTION
**Pull request by OrkhanAlikhanov**
Adding `@objc`s for #1247

### CODE REVIEW

**Comment by emauro** [comment]
Great! But is necessary to add to some other methods:

- init(rootViewController...

- isOpened

- isLeftViewOpened

- isRightViewOpened

**Comment by danieldahan** [comment]
@OrkhanAlikhanov I will wait to merge this in until all the necessary function declarations have been updated.

**Comment by OrkhanAlikhanov** [comment]
Hey @emauro! I added [`@objcMembers`](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html#ID593) to whole class instead of marking individual members objc.

> #### `@objcMembers`
>
>Apply this attribute to a class declaration, to implicitly apply the objc attribute to all Objective-C compatible members of the class, its extensions, its subclasses, and all of the extensions of its subclasses.
>
>Most code should use the objc attribute instead, to expose only the declarations that are needed. If you need to expose many declarations, you can group them in an extension that has the objc attribute. The objcMembers attribute is a convenience for libraries that make heavy use of the introspection facilities of the Objective-C runtime. Applying the objc attribute when it isn’t needed can increase your binary size and adversely affect performance.

**Comment by danieldahan** [comment]
@OrkhanAlikhanov Is this something we want to do to other classes across the board?

File: Sources/iOS/NavigationDrawer/NavigationDrawerController.swift [hunk 1]
```diff
@@ -132,6 +132,7 @@ public protocol NavigationDrawerControllerDelegate {
 }
 
 @objc(NavigationDrawerController)
+@objcMembers
```
Comment (danieldahan) [inline Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Hey! Can we add this feature to more classes and cleanup the single `@objc` declarations? As well to the other frameworks?

Comment (OrkhanAlikhanov) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: If @dorindanciu confirms that it works properly, we surely can!

Comment (danieldahan) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Cool, so let's wait for that. Thank you!

Comment (emauro) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: For me it is ok. @objc permits to fine grain the access to class methods but I think here it is not an issue.

Comment (danieldahan) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: @emauro so you tested it and it works?

Comment (emauro) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: @danieldahan Yes!

Comment (danieldahan) [reply Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Great, then we will add this where needed and make a new release. Thank you!

**Pull request merged**